guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-title-mobile@2x.jpg"
+   alt-text = "Photo of an ocean with the word 'Liquidity'"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+Editor's notes
+
+Figma file: https://www.figma.com/file/ZwRT4mZZ4UNGtsfLz22m8M/Liquidity?node-id=0%3A1
+
+-->
+
+# Liquidity
+
+Liquidity is a core concept to understand when working with the lightning network, although tricky. Ideally, we are able to design products that are easy enough to use so that users do not need to concern themselves with liquidity. However, product designers need to understand liquidity in order to build effectively on lightning.
+
+## What is liquidity?
+
+Put simply, liquidity on the lightning network is the ability to send or receive bitcoin. In the context of an individual user, liquidity is a measurement of their wallet's ability to send and receive bitcoin over lightning.
+
+### Inbound vs Outbound Liquidity
+
+*Inbound liquidity* is the amount of bitcoin that the user is able to receive over a lightning channel. *Outbound liquidity* is the amount of bitcoin that the user is able to send over a lightning channel.
+
+Receiving bitcoin decreases the user's inbound liquidity and increases their outbound liquidity. On the other hand, sending bitcoin decreases their outbound liquidity and increases their inbound liquidity.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-inbound-outbound-mobile@2x.png"
+   alt-text = "Graphic depicting inbound liquidity flowing towards a lightning node and outbound liquidity flowing away from a lightning node"
+   width = 800
+   height = 348
+%}
+
+{% include tip/open.html color="blue" icon="info" label="Sending and receiving" %}
+
+If your app offers the user visibility into the channel management process, you might consider using the term "receive limit" in place of "inbound liquidity" and "send limit" in place of "outbound liquidity".
+
+{% include tip/close.html %}
+
+## Why is liquidity important?
+
+Liquidity directly impacts the user experience. If a user has no outbound liquidity, they can not send bitcoin over lightning. Conversely, if a user has no inbound liquidity, they can not receive bitcoin over lightning.
+
+Sourcing inbound liquidity is very important for onboarding a user to a lightning wallet. Imagine if you were brand new to bitcoin. You download your first wallet, a lightning wallet. Because there is no inbound liquidity, you can not receive any bitcoin. Would you understand what to do next? Would the wallet still seem useful to you?
```

Comment:
```suggestion
Sourcing inbound liquidity is very important for [funding]({{'/guide/daily-spending-wallet/funding/' | relative_url}}) and onboarding a user to a lightning wallet. Imagine if you were brand new to bitcoin. You download your first wallet, a lightning wallet. Because there is no inbound liquidity, you can not receive any bitcoin. Would you understand what to do next? Would the wallet still seem useful to you?
```